compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/SymbolEnv.java

```diff
@@ -191,6 +192,13 @@ public static SymbolEnv getXMLAttributeEnv(BLangXMLAttribute node, SymbolEnv env
         return symbolEnv;
     }
 
+    public static SymbolEnv createStatementEnv(BLangStatement node, SymbolEnv env) {
```

Comment:
Generally a statement doesn't have an environment associated with it.
Since this is specific to a streaming query, shall we rename this to `createStreamingQueryEnv()` or something similar?
first argument can be `BLangStreamingQueryStatement`